src/admin.tsx

```diff
@@ -69,10 +109,12 @@ export default function AdminPage() {
     }
     newVideos[videoIndex] = newData;
 
-
     const docRef = doc(db, AMBIANCE_COLLECTION, documentId);
     return updateDoc(docRef, {
       videos: newVideos
+    }).then(() => {
+      setAlertSeverity("success");
+      handleAddSnackBarMessage("Successfully updated video - " + newData.name);
     });
```

Comment:
Do we want to add error handling to all of these in this PR? If so lets add an Error SnackBar message

Reply:
Done. I tested the error handling by temporarily changing Firestore rules to "false", not allowing any actions on the database.